components/clp-mcp-server/clp_mcp_server/server/session_manager.py

```diff
@@ -83,12 +83,18 @@ def cache_query_result_and_get_first_page(
         """
         :param results: Log entries from the query to cache.
         :return: Forwards `SessionState.get_page_data`'s return values.
-        :return: _GET_INSTRUCTIONS_NOT_RUN_ERROR if `get_instructions` has not been called in this
```

Comment:
if you removing this error message, will you be handling the case where `get_instructions` has not been called another way? or does that case no longer happen?